src/storage.h

```diff
@@ -54,6 +59,10 @@ namespace cycamore {
 /// Normal (optional)
 /// dormant_buying_mean is the mean length of the dormant buying period if dormant_buying_frequency_type is Normal
 /// dormant_buying_std is the standard deviation of the dormant buying period if dormant_buying_frequency_type is Normal
+/// dormant_buying_end_probability is the probability that at any given timestep, the agent ends the dormant buying period if
+/// the dormant buying frequency type is Binomial
+/// dormant_buying_disruption_probability is the probability that in any given cycle, the agent undergoes a disruption (disrupted
+/// offline period) if the dormant buying frequency type is FixedWithDisruption
```

Comment:
don't we need a `dormant_buying_disruption` entry?

Reply:
oops